### PR TITLE
Added Class Name for Static Methods

### LINKED ISSUE #18561: Static method calls are missing class name
In several places the sample code is missing class names in static method calls. For example it's `Console.WriteLine(...)`, not `WriteLine(...)`. And it's `SyntaxFactory.IdentifierName("System")` not `IdentifierName("System")`.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 442d453e-38dd-ed79-741c-813429a41822
* Version Independent ID: 5f1f64a1-9d23-0010-b505-d3874041f8f2
* Content: [Get started with syntax transformation (Roslyn APIs)](https://docs.microsoft.com/en-us/dotnet/csharp/roslyn-sdk/get-started/syntax-transformation)
* Content Source: [docs/csharp/roslyn-sdk/get-started/syntax-transformation.md](https://github.com/dotnet/docs/blob/master/docs/csharp/roslyn-sdk/get-started/syntax-transformation.md)
* Product: **dotnet-csharp**
* Technology: **csharp-roslyn**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by sackri10**
Fixed Issue - Static method calls are missing class name #18561

## Summary

Made changes to call statics methods with Class Name and removed the static in namespaces.

Fixes #18561 (if available)

### CODE REVIEW

**Comment by Youssef1313** [comment]
This comment (https://github.com/dotnet/docs/issues/18561#issuecomment-632704428) by @BillWagner states that the same issue exists in several samples.
So, I think the issue should still be open. I'd change Fixes to Contributes to or something to prevent closing it.

File: samples/snippets/csharp/roslyn-sdk/SyntaxTransformationQuickStart/ConstructionCS/Program.cs [hunk 1]
```diff
@@ -4,8 +4,8 @@
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 
 // <SnippetStaticUsings>
-using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
-using static System.Console;
+using  Microsoft.CodeAnalysis.CSharp;
+using  System;
 // </SnippetStaticUsings>
```
Comment (Youssef1313) [inline samples/snippets/csharp/roslyn-sdk/SyntaxTransformationQuickStart/ConstructionCS/Program.cs:9]: - You've extra space after `using`.
- The Snippet tag isn't meaningful now.
- The article that references this code will need to be updated as well to reflect the changes done in the snippet.

**Review by Youssef1313 — CHANGES_REQUESTED** [review]

**Review by BillWagner — APPROVED** [review]
This LGTM @sackri10 along with the additional changes in related PRs.

I'll merge them all today.  You should see the changes on the live site in the next day or two, on our regular publishing cycle.

Thanks again.

File: samples/snippets/csharp/roslyn-sdk/SyntaxTransformationQuickStart/ConstructionCS/Program.cs (hunk 1 above)
Comment (NextTurn) [reply samples/snippets/csharp/roslyn-sdk/SyntaxTransformationQuickStart/ConstructionCS/Program.cs:9]: Ping @BillWagner

**Pull request merged**